packages/shared-components/src/components/translate-label/translate-label.tsx

```diff
@@ -21,7 +21,16 @@ export class TranslateLabel {
 
   private readonly unsubscribeTranslationChanged: Function;
 
+  /**
+   * Represents a label key.
+   * @typedef {string} labelKey
```

Comment:
I think the typedef's are redundant here. Remove them

Reply:
done